docs/sources/datasources/aws-cloudwatch/aws-authentication.md

```diff
@@ -0,0 +1,111 @@
++++
+title = "Authentication"
+description = "AWS authentication"
+keywords = ["grafana", "aws", "authentication"]
+aliases = ["/docs/grafana/latest/datasources/cloudwatch"]
+weight = 205
++++
+
+# AWS authentication
+
+There are three different authentication methods available. 
```

Comment:
```suggestion
You can use one of three authentication methods: 
```

Reply:
@sunker, what is the fourth option? Can you enable it manually? Some details can help the user. Thanks.